### ISSUE TITLE
renders syntactically incorrect XSD datatype

### ISSUE THREAD

**Issue by jonassmedegaard** [sdm-lang/rust-sdml#9]
This in sdml source:
```
  @dc:created = @xsd:date("2024-04-26")
```

renders as this as turtle:
```
    dc:created "\"2024-04-26\""^^xsd:date ;
```

There is wrongly added escaped quotes.

The syntax is based on example 19 at https://sdml.io/reference/index.html

**Comment by johnstonskj** [comment]
Thanks, will take a look!

**Comment by johnstonskj** [comment]
Given the following SDML:

```
module type_constructor_fix9 is

  import dc

  @dc:description = "Regression test for bug #9"@en
  @dc:date = xsd:date("2024-06-27")

end
```

It generates the following Turtle:

```turtle
type_constructor_fix9:
    rdf:type
        owl:Ontology ,
        sdml:Module ;
    owl:imports <http://purl.org/dc/elements/1.1/> ;
    dc:description "Regression test for bug #9"@en ;
    dc:date "2024-06-27"^^xsd:date ;
    sdml:srcLabel type_constructor_fix9 .
```

**Comment by jonassmedegaard** [comment]
Thanks. I can confirm that this bug is now fixed (but possibly that very fix caused a regression: #11 ).
